Makefile

```diff
@@ -136,14 +133,13 @@ test-unit:
 # Build and vendor tarets
 #---------------------------------------------------------
 
-# Vendor target: "go mod vendor" resets the main module's vendor directory to include all packages needed to build and 
+.PHONY: build
+## Build: using operator-sdk to build a new image
+build:
+	$(Q)GO111MODULE=on operator-sdk build "$(GO_PACKAGE_ORG_NAME)/$(GO_PACKAGE_REPO_NAME):latest"
+
+# Vendor target: "go mod vendor" resets the main module's vendor directory to include all packages needed to build and
 # test all of the module's packages based on the state of the go.mod files and Go source code.
 ./vendor: go.mod go.sum
 	$(Q)GOCACHE=$(shell pwd)/out/gocache GO111MODULE=on go mod vendor ${V_FLAG}
 
-
-.PHONY: build
```

Comment:
We need this `build` target as it generates binaries of this operator and we use it in opeshift-ci. Am I right @baijum ? 
As `operator-sdk build` will give you an image so you can rename that target as `build-operator-image` or something similar and keep this target.

Reply:
Yes, this target is used for CI.

Reply:
I find this way of building a Golang project a bit more transparent instead of depending on a library :)

Reply:
As operator-sdk does not provide any significant way to produce binary of operator we need to use this way.